dom-manipulation/main.js

```diff
@@ -0,0 +1,25 @@
+var $button = document.querySelector('.hot-button');
+var $click = document.querySelector('.click-count');
+
+var i = 0;
```

Comment:
I would rename this to something like `numberOfClicks`. `i` isn't very descriptive of what this variable does